di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -243,7 +243,11 @@ public void doAuthorize(Context ctx) throws Exception {
     }
 
     public void apiAuthorize(Context ctx) throws Exception {
-        ctx.redirect(generateResponseRedirect(ctx.bodyAsClass(ApiAuthRequest.class)));
+        String redirectUrl = generateResponseRedirect(ctx.bodyAsClass(ApiAuthRequest.class));
+        JWTClaimsSet claimsSet = getClaimsSet(ctx.bodyAsClass(ApiAuthRequest.class));
```

Comment:
Slightly awkward that we decrypt the JAR twice - not massively significant but it is a relatively expensive crypto operation.

Perhaps you could call `getClaimsSet` here and pass it into `generateResponseRedirect` instead?